test/integration/clientset_test.go

```diff
@@ -164,11 +169,11 @@ func TestBrokerClient(t *testing.T) {
 
 	brokerServer.Spec.AuthSecret = authSecret
 
-	brokerUpdated, err := brokerClient.Update(brokerServer)
+	updated, err := client.Update(brokerServer)
```

Comment:
As I said above, a goal was to make this test easier on the eyes to read.  brokerClient isn't super bad, but serviceClassClient crosses some threshold where it just becomes absolutely horrible for my eyes to read (I normally wear contacts, but my eyes are pretty terrible).

I also think we will accumulate enough corner-cases to make it so that we'll want to split out the tests for each resource into their own files, at which point the qualification of names with the type of the resource becomes redundant.